### PR TITLE
Mks 48 create endpoint for entity

### PR DESCRIPTION
**Pull request by TsisinskyjV**

### CODE REVIEW

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -1,24 +1,56 @@
...
+using Nedo_net.Entities;
+using Nedo_net.Database;
+using System.Linq;
+using Newtonsoft.Json;
 
 namespace Nedo_net.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class ValuesController : ControllerBase
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/ValuesController.cs]: If you have controller for Student Entity, controller should be named StudentsController

Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/ValuesController.cs]: done

**Review by BohdanHamurda — APPROVED** [review]

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                return Ok(result);
+            }
+            catch (Exception ex)
+            {
+                return BadRequest(ex.Message);
+            }
+        }
+
+        // POST api/values
+        [HttpPost]
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs:57]: When are you going to implement BL(business logic) for other endpoints?

Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs:57]: in next task - 54 routing

Comment (bvzayats) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs:57]: ok, notice that you should have GetAll endpoint which returns existed data

**Review by bvzayats — APPROVED** [review]

**Pull request merged**